.appveyor.yml

```diff
@@ -1,21 +1,27 @@
+image: Visual Studio 2017
+
+clone_depth: 10
```

Comment:
:ok_hand: Didn't know of [shallow clones](https://git-scm.com/docs/git-clone#git-clone---depthltdepthgt), although I'm not sure to understand the implications...

Could this be an issue for a PR including more than 10 commits?

Reply:
> Could this be an issue for a PR including more than 10 commits?

No, I don't think it could be an issue. Even if the PR has more than 10 commits, we are only interested by tests on the last one, and this one will be here.

And I think it's the default option on travis.